### DISCUSSION TITLE
[Unstable] The detonator will "detonate" the explosives taken out from it after the countdown ends.

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#15301] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

As the title said

https://github.com/user-attachments/assets/9ccd4071-6e2e-42d9-ba0f-907b3edbb328

### Reproduction steps

1. Put a frag grenade inside a detonator
2. Activate the detonator and quickly take the grenade from it

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

Other

### -

v1.7.6.0 (Unsable)

### Which operating system did you encounter this bug on?

Windows

**Comment by zhu-rengong** [comment]
I fixed this in #15303 (along with optimization and modding support)

**Comment by Rokvach** [comment]
Tested the fix, working correctly. Fix coming in the next update. Closing discussion.
